src/components/navigation/desktop/DesktopNavigation.tsx

```diff
@@ -150,18 +150,17 @@ export const DesktopNavigation = () => {
     };
   }, []);
 
-  async function clearAnalytics(event: any) {
-    recordClick(event);
+  async function clearAnalytics() {
     await flushEvents();
   }
```

Comment:
I think we can call `flushEvents` without `await` and totally remove this function @calebjacob WDYT?

Reply:
@shelegdmitriy Right, we can probably get rid of `clearAnalytics()` entirely. I'm guessing it was originally added to help with testing the events when first being developed.